### ISSUE TITLE
LAMB optimizer instead of SGD!

### ISSUE THREAD

**Issue by srijandas07** [CupidJay/MoCov3-pytorch#3]
I guess the optimizer choice should be either LARS or LAMB based on the encoder (convolutional/transformer). Isn't it?

**Comment by CupidJay** [comment]
Yes,  I think so. Since MoCov3 requires a relative large batch size, so LARS is a better choice. Please refer to [https://github.com/facebookresearch/simsiam/blob/main/main_lincls.py#L232](url) or [https://github.com/facebookresearch/barlowtwins/blob/main/main.py#L224](url) for specific implementation.
